telephoneNumberValidator/script.js

```diff
@@ -0,0 +1,40 @@
+const form = document.getElementById('my-form');
+let userInput = document.getElementById('user-input');
+let result = document.getElementById('is-palindrome');
+
+const TIMEOUT = 3000;
+
+function telephoneCheck(str) {
+    let regex = /^1{0,1}\s*(\(\d{3}\)|\d{3})(\s*|-)\d{3}(\s*|-)\d{4}$/;
+
+    if (regex.test(str)) return true;
+    
+    return false;
+}
+
+function timer() {
+    return setTimeout(() => {
+        result.textContent = '';
+        result.style.color = 'black';
+        result.classList.remove('move');
+    }, TIMEOUT)
+}
+
+function showResult(phoneNumber) {
+    if (phoneNumber) {
+        result.textContent = `${userInput.value} is valid telephone number!`;
+        result.style.color = 'green';
```

Comment:
see above. And please check your whole code for such things

Reply:
fixed